/*
 * Copyright (c) 2019 - now, Eggroll Authors. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *
 *
 */

syntax = "proto3";

package com.webank.eggroll.core.transfer;

message GetLogRequest {
  string sessionId = 1;
  string rank = 2;
  string path = 3;
  int32  startLine = 4;
  string logType =5;
}

message GetLogResponse {
  repeated string datas = 2;
}


// TODO: use transfer lib
service ExtendTransferServer {
  rpc getLog ( stream GetLogRequest) returns (stream GetLogResponse);
}
